### ISSUE TITLE
Pagination colors are incorrect

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/generator-cf#129]
Paginator has a link (the Go link) on a gray-10 background which is not accessible. This should be gray-5.

## Current behavior

- https://github.com/cfpb/generator-cf/blob/master/app/templates/src/static/css/cf-theme-overrides.less#L212

## Expected behavior

- Should be gray-5 for paginator background and disabled button background.

**Comment by jimmynotjim** [comment]
We can completely remove the brand-palette.less file right @Scotchester?

**Comment by Scotchester** [comment]
Yes, along with `cf-theme-overrides.less` (or empty it out) and `licensed-fonts.less`. Palette and fonts now come with cf-core and cf-typography, respectively.
